.github/workflows/chart-testing.yaml

```diff
@@ -62,4 +62,8 @@ jobs:
       with:
         version: 'latest'
     - name: Run chart-testing (install)
-      run: ct install --config chart-testing.yaml
+      run: ct install --config chart-testing.yaml --helm-extra-set-args '--set=exporter.tls.disabled=true --set=auth0.domain="${{ secrets.TEST_AUTH0_DOMAIN }}" --set=auth0.clientId="$CLIENT_ID" --set=auth0.clientSecret="$CLIENT_SECRET"'
```

Comment:
You can also create `ci/test-values.yaml` in the root of the chart directory and `chart-testing` will use those when it installs. The format is just regular helm values, but only overrides from default values.yaml.

Reply:
If I had a `ci/test-values.yaml` is there a way I can populate the values in the yaml with the secrets in CI?
Maybe I could use `envsubst`, but I'm not 100% sure

Reply:
Oh I missed that bit - no. The secrets would need to get written as a k8s secret and then referenced in the chart.